docs/operator-manual/applicationset/Progressive-Rollouts.md

```diff
@@ -0,0 +1,106 @@
+# Progressive Rollouts
+
+This is an experimental, alpha-quality feature that allows you to control the order in which the ApplicationSet controller will create or update the Applications owned by an ApplicationSet resource. It may be removed in future releases.
```

Comment:
```suggestion
!!! warning "Alpha Feature"
    This is an experimental, alpha-quality feature that allows you to control the order in which the ApplicationSet controller will create or update the Applications owned by an ApplicationSet resource. It may be removed in future releases or modified in backwards-incompatible ways.
```